### PR TITLE
Created user object to be used as an entity type. 

### PR DESCRIPTION
**Pull request by lattai**
Included getters and setters for all attributes including email, displayName, imageID, and currentPageID.

### CODE REVIEW

**Comment by lzoebaker** [comment]
It seems like there is a compilation/ build error? @lattai

**Comment by menynexi** [comment]
if you look at the build error you will see it failed in the maven so you have to go to the pom.xml and add it

File: src/main/java/com/google/sps/data/SiteUser.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.appengine.api.datastore.Key;
+
+public class SiteUser {
```
Comment (WillHipschman) [inline src/main/java/com/google/sps/data/SiteUser.java:17]: Why SiteUser and not just User? 

I would also suggest something a bit more focused like "Player"

File: src/main/java/com/google/sps/data/SiteUser.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    private String displayName;
+    private String email;
+    private Key key;
```
Comment (WillHipschman) [inline src/main/java/com/google/sps/data/SiteUser.java:20]: What is the key for? Can you name the variable more clearly?

File: src/main/java/com/google/sps/data/SiteUser.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    private String imageID;
```
Comment (WillHipschman) [inline src/main/java/com/google/sps/data/SiteUser.java:21]: Does it make sense for these Ids to be GUIDs? Are they values generated/returned from the database?

**Pull request merged**